packages/search-ui-analytics-plugin/package.json

```diff
@@ -0,0 +1,41 @@
+{
+  "name": "@elastic/search-ui-analytics-plugin",
+  "version": "1.17.1",
```

Comment:
I guess we don't really have the ability to release this as a pre-release or anything?

Reply:
Yeh we dont have the ability yet for a pre-release without difficulty.